pkg/gofr/gofr.go

```diff
@@ -88,6 +88,17 @@ func New() *App {
 	app.httpServer.certFile = app.Config.GetOrDefault("CERT_FILE", "")
 	app.httpServer.keyFile = app.Config.GetOrDefault("KEY_FILE", "")
 
+	// Add Default routes
+	app.add(http.MethodGet, "/.well-known/health", healthHandler)
+	app.add(http.MethodGet, "/.well-known/alive", liveHandler)
+	app.add(http.MethodGet, "/favicon.ico", faviconHandler)
+
+	if _, err = os.Stat("./static/openapi.json"); err == nil {
+		app.add(http.MethodGet, "/.well-known/openapi.json", OpenAPIHandler)
+		app.add(http.MethodGet, "/.well-known/swagger", SwaggerUIHandler)
+		app.add(http.MethodGet, "/.well-known/{name}", SwaggerUIHandler)
```

Comment:
This line requires a comment

Is it a kind of catchall to send anything that is not openapi.json and swagger to the SwaggerUIHandler

Reply:
Sure added the comments